bundles/org.openhab.binding.androidtv/README.md

```diff
@@ -30,37 +32,53 @@ This binding requires GoogleTV to be installed on the device (https://play.googl
 
 ## Thing Configuration
 
-There are three required fields to connect successfully to a ShieldTV.
+The is one required field to connect to the devices.  All other fields are optional.
 
 | Name             | Type    | Description                           | Default | Required | Advanced |
 |------------------|---------|---------------------------------------|---------|----------|----------|
 | ipAddress        | text    | IP address of the device              | N/A     | yes      | no       |
 | googletvPort     | text    | TCP Port for GoogleTV                 | 6466    | no       | no       |
 | shieldtvPort     | text    | TCP Port for ShieldTV                 | 8987    | no       | no       |
+| philipstvPort    | text    | TCP Port for PhilipsTV                | 1926    | no       | no       |
 | keystore         | text    | Location of the Java Keystore         | N/A     | no       | no       |
 | keystorePassword | text    | Password of the Java Keystore         | N/A     | no       | no       |
 | gtvEnabled       | boolean | Enable/Disable the GoogleTV protocol  | true    | no       | no       |
```

Comment:
This table does not reflect all changes. Some parameters to GoogleTV have been added in the thing-type.xml and missinghere, same for PhilipsTV.

As the Things have a slightly different config, it would also be usefull to somehow document what parameteris applicalbe for a specific Thing.

Reply:
I think I fixed all of this in both thing-type and readme.  Please give it a double check.  Somehow the rebase got messed up on the xml file in here.

Reply:
lgtm